core/src/main/java/org/eclipse/hono/util/LinkHelper.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2016, 2018 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
```

Comment:
We changed the header to a more generic form ("...Contributors to the Eclipse Foundation").
See https://www.eclipse.org/hono/community/contributing/ for details.
Please adapt to this.